### ISSUE TITLE
POST /login problem

### ISSUE THREAD

**Issue by herikudaru** [Arcue/Webshop_userDatabase#13]
Dokumentation säger att man bör skicka över email och password i json för att få token. För tillfället fungerar det inte, istället om man skickar namn och password fungerar det.

Fungerar inte:
```
{  
   "email":"email@email.com",
   "password":"password"
}
```

Fungerar:
```
{  
   "email":"namn",
   "password":"password"
}
```

**Comment by JolleJ** [comment]
> Dokumentation säger att man bör skicka över email och password i json för att få token. För tillfället fungerar det inte, istället om man skickar namn och password fungerar det.
> 
> Fungerar inte:
> 
> ```
> {  
>    "email":"email@email.com",
>    "password":"password"
> }
> ```
> Fungerar:
> 
> ```
> {  
>    "email":"namn",
>    "password":"password"
> }
> ```

yes det är email som skall skickas för att få token, ändrar det så snabbt jag kan
